### PR TITLE
Update latest version and latest news

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

File: _includes/version.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="version-marker">
   <p>
     Latest versions:
+      <a href="{{ site.baseurl }}manuals/2.5/index.html#Releasenotesfor2.4">2.5.0</a>
```
Comment (ekohl) [inline _includes/version.html]:
```suggestion
      <a href="{{ site.baseurl }}manuals/2.5/index.html#Releasenotesfor2.5">2.5.0</a>
```

**Review by ekohl — CHANGES_REQUESTED** [review]
I think you have missed one instance.

File: _includes/version.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+      /
       <a href="{{ site.baseurl }}manuals/2.4/index.html#Releasenotesfor2.4">2.4.0</a>
       /
       <a href="{{ site.baseurl }}manuals/2.3/index.html#Releasenotesfor2.3">2.3.4</a>
```
Comment (upadhyeammit) [inline _includes/version.html:8]: Not sure if I should remove 2.3.4 as we decided to do 2.3.5

Comment (ekohl) [reply _includes/version.html:8]: Since we're going to do a 2.3.5, I'd keep it in for now.

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**